### PR TITLE
Add: social bar section

### PR DESCRIPTION
**Pull request by gergana95**
screen shot 2017-07-11 at 4 50 28 pm

Added: social bar section html syntax and css styling, fixed alignment.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -155,6 +156,51 @@
...
+    margin-bottom: 0;
+}
+
+@keyframes colorchange {
+    from { fill: #ebebeb; }
+    to { fill: #5da8ef; }
+}
+.social-icons__icon:hover > path {
+    animation-name: colorchange;
+    animation-duration: 4s;
```
Comment (superKalo) [inline index.css]: Cool animation, but we need a `transition` here instead.

File: index.css [hunk 2]
```diff
@@ -155,6 +156,51 @@
...
+
+.social-icons {
+    padding-top: 285px;
+}
+
+.social-icons > a {
+    text-decoration: none;
+    display: block;
+    margin-bottom: 23px;
+}
```
Comment (superKalo) [inline index.css]: Let's discuss why "> a" is kind of a bad idea here.

**Comment by superKalo** [comment]
1. Body horizontal scroll -> overflow: hidden;
2. For the little 90deg-aligned-label: https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

File: index.css [hunk 3]
```diff
@@ -155,6 +156,51 @@
...
+    font-weight: 700;
+    letter-spacing: 6px;
+    -ms-transform: rotate(180deg); /* IE 9 */
+    -webkit-transform: rotate(180deg); /* Chrome, Safari, Opera */
+    transform: rotate(180deg);
+    writing-mode: vertical-lr;
+}
+
+.social-icons {
+    padding-top: 285px;
```
Comment (superKalo) [inline index.css]: Do it in a different way.

File: index.css (hunk 1 above)
Comment (gergana95) [reply index.css]: changed with transition

File: index.css (hunk 3 above)
Comment (gergana95) [reply index.css]: made in a better way

File: index.css (hunk 2 above)
Comment (gergana95) [reply index.css]: added new class

**Comment by superKalo** [comment]
Looks good to me 👍

**Pull request merged**